### PR TITLE
Trans safety network

### PR DESCRIPTION
**Pull request by honor-gfsc**
## Description

- Added draft of TSN to projects
- Copy would need approval from someone who worked directly on the project before publishing, so for this pull the project page is still marked as a draft. 

@geeksforsocialchange/developers

### CODE REVIEW

File: content/project/trans-safety-network/index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+date: 2022-05-01
+title: "Trans Safety Network"
+client: "Trans Safety Network"
+when: "Autumn 2022"
+who: kim mark
+themes: trans-lib
+categories: consultancyresearch designbranding
```
Comment (kimadactyl) [inline content/project/trans-safety-network/index.md:8]: add software tag

File: content/project/trans-safety-network/index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+date: 2022-05-01
+title: "Trans Safety Network"
```
Comment (kimadactyl) [inline content/project/trans-safety-network/index.md:3]: Trans Safety Network Website and Rebrand?

File: content/project/trans-safety-network/index.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+client: "Trans Safety Network"
+when: "Autumn 2022"
```
Comment (kimadactyl) [inline content/project/trans-safety-network/index.md:5]: Should we make it like "Autumn-Winter 2022"? I think we launched in Jan?

**Comment by kimadactyl** [comment]
Apols forgot about this - just a few tweaks to metadata and i think good to go, looks good!

**Comment by honor-gfsc** [comment]
@kimadactyl sorry, I don't really understand Github fully. Did you *make* the changes, or do I need to do it on my version?

**Comment by kimadactyl** [comment]
Yeah these are suggested changes - you can reply to the specific suggestions, or just make changes and push if you agree

**Pull request merged**